### PR TITLE
[+] updated to react 0.14.0

### PR DESCRIPTION
**Pull request by eyala-stratoscale**

### CODE REVIEW

**Comment by emmenko** [comment]
Thanks!

One thing to adjust regarding the owner context
- you don't need `bind` anymore
  https://github.com/emmenko/redux-react-router-async-example/blob/master/lib/Root.js#L44

**Comment by eyala-stratoscale** [comment]
I'm looking through v0.14 docs, couldn't find anything regarding it.
What should this line be instead of :  {renderRoutes.bind(null, props.history)}  ?

**Comment by emmenko** [comment]
Just `renderRoutes(props.history)` :)

**Comment by eyala-stratoscale** [comment]
Updated, have a look

**Comment by emmenko** [comment]
:+1: 

Can you confirm that everything works fine? Then I can merge :)

**Comment by eyala-stratoscale** [comment]
Hey,
When I checked it yesterday it seemed ok, but I'm quite new to this and
don't know your code.
It's better that you have a look when you have time.

File: lib/components/MenuListItem.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import React, { PropTypes } from 'react'
+import React, { PropTypes, Component } from 'react'
 import { Link } from 'react-router'
 
-export default class MenuListItem {
+export default class MenuListItem extends Component{
```
Comment (emmenko) [inline lib/components/MenuListItem.js]: Space before the bracket `Component {`

**Comment by emmenko** [comment]
Fix the lint, then it's good to merge (just tested it locally).

**Comment by eyala-stratoscale** [comment]
fixed

**Comment by emmenko** [comment]
:+1:  thanks for the PR!

**Comment by eyala-stratoscale** [comment]
you welcome, thanks for the repo

**Pull request merged**